module-2/visualizing-real-world-data/your-code/your-code.ipynb

```diff
@@ -1253,22 +1253,22 @@
   },
```

Comment:
- FYI: when we want to run bining there are some general rules we should follow in order to get the proper insights from the data. 
1. Bins should be all the same size. For example, groups of ten or a hundred.
2. Bins should include all of the data, even outliers. If your outliers fall way outside of your other data, consider lumping them in with your first or last bin. This creates a rough histogram —make sure you note where outliers are being included.
3. Boundaries for bins should land at whole numbers whenever possible (this makes the chart easier to read).
Choose between 5 and 20 bins. The larger the data set, the more likely you’ll want a large number of bins. For example, a set of 12 data pieces might warrant 5 bins but a set of 1000 numbers will probably be more useful with 20 bins. The exact number of bins is usually a judgment call.
4. If at all possible, try to make your data set evenly divisible by the number of bins. For example, if you have 10 pieces of data, work with 5 bins instead of 6 or 7.
- Given the characteristics of your dataset I would have split the analysis based on price range. As you have enough number of records, you could have create a subset and run the plots in each subset. Maybe you could have got more insights. Just an idea for future projects.
- User rating is a discrete variable similar to a categorical variables. This is the reason because your scatter plot does not throw any valuable info.